core/server/models/user.js

```diff
@@ -783,6 +783,10 @@ User = ghostBookshelf.Model.extend({
                 return self.isPasswordCorrect({plainPassword: object.password, hashedPassword: user.get('password')})
                     .then(function then() {
                         return user.updateLastSeen();
+                    })
+                    .then(function then() {
+                        user.set({status: 'active'});
+                        return user.save();
```

Comment:
We _could_ have done smth like
```javascript
user.set({status: 'active'})
return user.updateLastSeen()
```
But that relies too much on the functionality of `updateLastSeen` and seemed brittle

Reply:
Hm yeah agree, that's not cool